pkg/operators/wasm/wasm.go

```diff
@@ -230,6 +237,7 @@ func (i *wasmOperatorInstance) init(
 	i.addPerfFuncs(env)
 	i.addKallsymsFuncs(env)
 
+	cc.refreshMu.Lock()
```

Comment:
1. What is lock protecting from? (cache.Close() being called before this module is initialized?)
2. It's never released if any of the returns below are executed 
3. It seems it's covering a big zone. Is it possible to use a rw lock to avoid multiple "wasm" modules being compiled in parallel?